aquacomputer_d5next.c

```diff
@@ -2506,6 +2515,8 @@ static int aqc_raw_event(struct hid_device *hdev, struct hid_report *report, u8
 	if (report->id != STATUS_REPORT_ID)
 		return 0;
 
+	print_hex_dump_bytes("status_report: ", DUMP_PREFIX_OFFSET, data, size);
```

Comment:
This is of course a no-op unless debugging is enabled through static or dynamic debugging.  However it was incredibly useful as a tool for...debugging.